terraform/service_exposure.tf

```diff
@@ -76,15 +77,15 @@ resource "google_cloud_run_service" "exposure" {
 
     metadata {
       annotations = {
-        "autoscaling.knative.dev/maxScale" : "1000",
-        "run.googleapis.com/cloudsql-instances" : google_sql_database_instance.db-inst.connection_name
+        "autoscaling.knative.dev/maxScale" : "50",
```

Comment:
that's only 4000 concurrent requests total. Which is a potential risk due to the 5s latency hold.

Trying to decide if I want it changed or not though.

Reply:
I updated the database max_connections to 100k and changed this to 500 concurrent (40k total conns)